static/src/stylesheets/module/commercial/glabs/_hosted-video.scss

```diff
@@ -134,6 +134,17 @@
     pointer-events: none;
 }
 
+.youtube__video-buffering,
+.youtube__video-playing,
+.youtube__video-paused {
+    ~ .hosted__youtube-poster-image {
+        // HACK! "hide" the poster image (with a 0 height) when video is playing
+        // but position static for hosted__youtube-play-button's absolute positioning to work
+        height: 0;
+        position: static;
```

Comment:
this element has a child which needs to be shown on video pause, `display: none` effects child elements too